lib/kafka/broker_pool.rb

```diff
@@ -2,12 +2,14 @@
 
 module Kafka
   class BrokerPool
-    def initialize(client_id:, connect_timeout: nil, socket_timeout: nil, logger:)
+    def initialize(client_id:, connect_timeout: nil, socket_timeout: nil, logger:, ssl: nil, ssl_context: nil)
```

Comment:
I'm not sure about this API – why not just have e.g. `ssl_context` and disable SSL if it's nil?